### ISSUE TITLE
Point cloud process 

### ISSUE THREAD

**Issue by RenFeiTemp** [rusty1s/pytorch_geometric#177]
## ❓ Questions & Help

Hi, 
I tried to use the default pytorch dataloader to load point clouds. I have reshaped the point tensors to [B * N, 3] and create a batch-vector [B*N] and feed them to the pointnet++. 
I get error at 
x = F.relu(self.local_sa1(None, pos, edge_index))
as follows:
File "build/bdist.linux-x86_64/egg/torch_geometric/nn/conv/point_conv.py", line 46, in forward
File "build/bdist.linux-x86_64/egg/torch_geometric/nn/conv/message_passing.py", line 86, in propagate
File "build/bdist.linux-x86_64/egg/torch_geometric/utils/scatter.py", line 33, in scatter_.

Thanks in advance.

**Comment by rusty1s** [comment]
Are you sure there is no other error message? This is kinda meaningless. It would help if you could print `pos` and `edge_index` size, dtype, and device before calling `local_sa1`.

**Comment by RenFeiTemp** [comment]
There are the information of those tensor and the missing error message
('pos', (32768, 3), torch.float32, device(type='cuda', index=0))
('edge_index', (2, 576249), torch.int64, device(type='cuda', index=0))
THCudaCheck FAIL file=/opt/conda/conda-bld/pytorch_1549614443593/work/aten/src/THC/generated/../THCTensorMathCompare.cuh line=82 error=11 : invalid argument

**Comment by rusty1s** [comment]
This seems to be an indexing problem. Can you run again with `CUDA_LAUNCH_BLOCKING=1`?

Additionally, please report the output of
```python
print(edge_index.min().item(), edge_index.max().item())
```

**Comment by RenFeiTemp** [comment]
When i reduce the range of the radius, the size of edge_index reduce to (2, 117343) and the code is work

**Comment by rusty1s** [comment]
Ah, I see. You can alternatively increase the `max_neighbors` arg in the radius call. However, this should not crash and I will fix it.

**Comment by rusty1s** [comment]
Fixed in master and `v1.1.1`! Thank you.

**Problem:**
The `radius` call expects a `max_num_neighbors` argument, which may result in ignoring a lot of points when the range of the radius is set to a large value. Hence, a bipartite graph is constructed where possibly not all nodes have an edge to a sampled node. Before this bugfix, we took an initial guess of the bipartite graph size (given by `edge_index[0].max()` and `edge_index[1].max()`), which could yield a wrong size for reasons mentioned above. Hence, the `PointConv` does now additionally expect a `size` argument which takes in the adjacency matrix size as a tuple. If set to `None`, its size is given by the initial guess.
